src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,29 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = (Logger) LogManager.getLogger(OrderServiceImpl.class);
```

Comment:
Casting `LogManager.getLogger(OrderServiceImpl.class)` to `Logger` is unnecessary. You can directly assign it to `logger` without casting.